Makefile

```diff
@@ -38,66 +36,83 @@ GOLANGCI_LINT_VERSION := 1.61.0
 
 export PATH := $(PATH):${GOBIN}
 
+.PHONY: all
 all: tools test validate .gitvalidation
 
+.PHONY: build
 build:
 	go build $(BUILDFLAGS) ./...
 
 $(MANPAGES): %: %.md
 	$(GOMD2MAN) -in $< -out $@
 
+.PHONY: docs
```

Comment:
This one is not actually phony AFAICS: it’s a set of file dependencies with correct file dependencies on the sources.